app/modules/ads/ml/tb-viewability.js

```diff
@@ -0,0 +1,73 @@
+let config = {};
```

Comment:
Why not create a class and store configuration in `config` property?

Reply:
it's been dealt with differently